ejb3/src/main/java/org/jboss/as/ejb3/component/stateful/StatefulSessionComponent.java

```diff
@@ -203,11 +201,11 @@ public EJBObject getEJBObject(final InterceptorContext ctx) throws IllegalStateE
             return AccessController.doPrivileged(new PrivilegedAction<EJBObject>() {
                 @Override
                 public EJBObject run() {
-                   return EJBClient.createProxy(new StatefulEJBLocator<EJBObject>((Class<EJBObject>) view.getViewClass(), locatorAppName, getModuleName(), getComponentName(), getDistinctName(), getSessionIdOf(ctx), getCache().getStrictAffinity(), WildFlySecurityManager.getPropertyPrivileged(ServerEnvironment.NODE_NAME, null)));
+                   return EJBClient.createProxy(new StatefulEJBLocator<EJBObject>((Class<EJBObject>) view.getViewClass(), locatorAppName, getModuleName(), getComponentName(), getDistinctName(), getSessionIdOf(ctx), getCache().getStrictAffinity()));
```

Comment:
This should be made non-"unsafe" at some point.  Not really important for now though.